.travis.yml

```diff
@@ -24,6 +24,8 @@ matrix:
       env: DEPS=dev SYMFONY_VERSION='3.4.*'
     - php: 7.1
```

Comment:
Might be time to consider dropping 7.1 and below? They are all EOL

Reply:
Good idea, will do in another PR.